x/margin/keeper/keeper.go

```diff
@@ -534,6 +539,106 @@ func (k Keeper) GetMTP(ctx sdk.Context, mtpAddress string, id uint64) (types.MTP
 	return mtp, nil
 }
 
+func (k Keeper) GetMTPIterator(ctx sdk.Context) sdk.Iterator {
+	store := ctx.KVStore(k.storeKey)
+	return sdk.KVStorePrefixIterator(store, types.MTPPrefix)
+}
+
+func (k Keeper) GetAllMTPS(ctx sdk.Context) []types.MTP {
```

Comment:
@kenta-elys very minor but might want to rename that to `GetAllMTPs`

Reply:
@cosmic-vagabond Done.